tests/mocks/eth_account.cairo

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Contracts for Cairo v0.1.0 (account/Account.cairo)
```

Comment:
```suggestion
# OpenZeppelin Contracts for Cairo v0.1.0 (account/EthAccount.cairo)
```
I think this should be a preset instead of a mock. I'd call it `EthAccount`